src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs

```diff
@@ -157,11 +189,17 @@ internal void ReadExtensionMembers(ref MessagePackReader reader)
     }
 
     /// <summary>
-    /// Interface of ack-able message 
+    /// The ASRS send it to the source server connection to initiate a client connection migration.
+    /// Indicates that incoming messages are blocked.
     /// </summary>
-    public interface IAckableMessage
+    /// <remarks>
+    ///
+    /// </remarks>
+    /// <param name="connectionId">The client connection ID</param>
+    /// <param name="state">The state of control message</param>
+    public class FlowControlMessage(string connectionId, FlowControlState state) : ConnectionMessage(connectionId)
```

Comment:
Do we need Connection prefix? `ConnectionFlowControlMessage` since connectionId is a required parameter?